ajax/libs/qcobjects/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "qcobjects",
+  "version": "2.1.298",
+  "description": "QCObjects a Quickcorp Objects Library - Cross Browser Javascript Framework for MVC Patterns",
+  "main": "qcobjects.min.js",
```

```suggestion
  "filename": "qcobjects.min.js",
```
This attr should be named `filename`, not `main`.

---

Done.